lab-03-chris/index.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const fsJs = require('./script/lib/fs-read.js');
+
```

Comment:
just because they're called in order, doesn't mean they'll return the way you'd expect. that's why we will use promises and nested callbacks - to ensure asynchronous execution.